settings.gradle

```diff
@@ -5,3 +5,5 @@ include ':opensrp-goldsmith'
 /*include ':opensrp-app'
 project(":opensrp-app").projectDir = new File("../opensrp-client-core/opensrp-app")*/
 
+include ':opensrp-chw'
+project(":opensrp-chw").projectDir = new File("../opensrp-client-chw/opensrp-chw")
```

Comment:
Do we need CHW currently?

Reply:
No. This was there for reference. I'll remove it